### ISSUE TITLE
"Updating failed. Could not delete meta value from database."

### ISSUE THREAD

**Issue by MadtownLems** [markjaquith/page-links-to#127]
I'm finally getting around to exploring moving to the Block Editor, and I ran into the following problem when testing Page Links To.

"Updating failed. Could not delete meta value from database."

Steps to reproduce the issue:

1) Edit an existing page that is not currently using Page Links To.
2) Click the radio for "A custom URL"
3) Click the radio for "Its normal WordPress URL"
4) Click 'Update'

The page will not update, and the aforementioned error is displayed at the top of the screen.

I've also experienced the error in more "real world" situations, but the above seems the simplest to have it happen all the time. I'm hoping that resolving this easy-to-reproduce case also solves the other times it's been appearing for me. 

(WordPress 5.5.3, no other plugins.)

**Comment by MadtownLems** [comment]
I'm typically pretty good at helping sort these issues out with other developers, but unfortunately, we're just getting into the Gutenberg stuff (hence how I even stumbled on this).

This is perhaps a little more info about the error (the response from the ajax call when trying to publish in the situation described again):

{"code":"rest_meta_database_error","message":"Could not delete meta value from database.","data":{"key":"_links_to","status":500}}

In short, it seems like it fails when it tries to delete a meta value for '_links_to' when it doesn't exist. So, wherever this is happening, perhaps you can simply confirm that that meta_value actually exists before attempting to delete it?

Thanks & Cheers

**Comment by dethfactor** [comment]
Running into the same issue for a client. Any ETA on a resolution?

**Comment by jenfloods** [comment]
I have this issue as well, using Gutenberg Block Editor (we also use Divi Builder but not on Posts?). The error for me is: "Updating failed. Could not update the meta value of _et_pb_use_builder in database". How do we resolve this issue?
